test/extended/router/gatewayapicontroller.go

```diff
@@ -31,6 +36,12 @@ var (
 		configv1.ClusterVersionCapabilityOperatorLifecycleManager,
 	}
 )
+const (
+	// Max time duration for the DNS resolution
+	dnsResolutionTimeout = 10 * time.Minute
+	// Max time duration for the Load balancer address
+	LoadBalancerReadyTimeout = 10 * time.Minute
```

Comment:
Since some changes will have to be done anyway. Can we add this to the PR too?
```suggestion
	loadBalancerReadyTimeout = 10 * time.Minute
```
We don't need this variable to be exported off the package.

Reply:
done